includes/parse_md.php

```diff
@@ -101,7 +104,11 @@ function parse_md($markdown){
   }
 
   // Find and replace emojis names with images
-  $content = preg_replace('/:([\S]+?):/','<img class="emoji" alt="${1}" height="20" width="20" src="https://github.githubassets.com/images/icons/emoji/${1}.png">',$content);
+  $content = preg_replace('/:(?!\/)([\S]+?):/','<img class="emoji" alt="${1}" height="20" width="20" src="https://github.githubassets.com/images/icons/emoji/${1}.png">',$content);
 
-  return $content;
+  $output['content'] = $content;
+  $output["meta"] = $meta;
+  $output["title"] = $title;
+  $output["subtitle"] = $subtitle;
+  return $output;
```

Comment:
NB: This breaks a couple of other pages - namely `docs_schema.php` and `launch.php` (easy fix).

Reply:
Fix for docs_schema.php in dcdfddb.  `launch.php` has it's own parse_md(). Should we replace it with the global function?

Reply:
Ah, yeah that sounds sensible. Sorry, reinventing the wheel again there. I just found it on a project text search, didn't realise that it was different but named identically 🤦🏻